docs/compatibility.md

```diff
@@ -1,5 +1,7 @@
 # Compatibility
 
+## Executable "apps"
```

Comment:
Should we use the terminology "projects" around this since the differentiator for this list and the following one is whether it's a project or not as I understand.

Reply:
The use of "app" here means the project is executable in some way. Everything is a project. Would it help to say `Executable "app" projects` instead?